src/extlibs/hrvo/velocity_obstacle.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "software/geom/vector.h"
+
+/**
+ * A hybrid reciprocal velocity obstacle.
+ */
+class VelocityObstacle
```

Comment:
why do we need a separate data type?

Reply:
Had to move it to another file to avoid a circular dependency (as it was originally defined in Agent.h which is a part of the HRVO bazel library)